recipes/cctbx/meta.yaml

```diff
@@ -0,0 +1,97 @@
+{% set version = "2020.8" %}
+
+package:
+  name: cctbx-base
+  version: {{ version }}
+
+source:
+  url: https://github.com/cctbx/cctbx_project/releases/download/v{{ version }}/cctbx-{{ version }}.tar.gz
+  sha256: 38b41bb5e3b968e499a35c30de81147abb946c8de2728e90676f24843348b033
+  patches:
+    - crys3d.patch
+    - cbf.patch  # [win]
+    - libtbx.patch
+
+build:
+  number: 0
+  missing_dso_whitelist:
+    - '*/libGL.so.1'  # [linux]
+    - '*/libGLU.so.1'  # [linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - boost >=1.65
```

Comment:
You shouldn't set a version constraint here as it will come from the global pinning. As it stands it will always build against the newest boost which will make conflicts very likely.

Unrelated, are you using boost python? If not you should use boost-cpp here.

Reply:
Yes, we are using Boost.Python to expose C++ code to Python. And just committed a change.